### PR TITLE
chore: Show error when visiting from a `file://` uri.

### PR DESCRIPTION
**Pull request by ArjixWasTaken**

### CODE REVIEW

**Comment by ArjixWasTaken** [comment]
Since the other split PRs kinda depend on this one (due to the file structure change), I'll open the other 2 PRs after this PR is merged.

(if it is merged, otherwise I'll undo these changes in the other two PRs)

**Comment by rexim** [comment]
Wait, what?! How on Earth displaying numbers on a canvas depends on... anything in fact? Am I missing something?

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 all: wasm/game.wasm
 
 wasm/game.wasm: game.c
-	clang --target=wasm32 -I./include --no-standard-libraries -Wl,--no-entry -Wl,--allow-undefined -Wl,--export=game_init -Wl,--export=game_frame -Wl,--export=game_over -o wasm/game.wasm game.c -DPLATFORM_WEB
```
Comment (rexim) [inline Makefile:4]: No, I like the name `wasm` better, cause `assets` just invites you throw all the crap in it.

**Comment by rexim** [comment]
I'm very conflicted on this PR tbh... Is this really that important? We are making a thing for programmers who understand what they are doing. The whole thing is expected to be simple and hackable. I dunno... I'll need to think about it.

**Comment by rexim** [comment]
Alright, I think it's all fine as long as it's outside of raylib.js. Please, rename `wasm` folder back and I'll merge this PR. Thank you! :+1:

**Comment by ArjixWasTaken** [comment]
> Wait, what?! How on Earth displaying numbers on a canvas depends on... anything in fact? Am I missing something?

I meant that I depend on the custom font, which would be under the `assets` directory.
But I can have that changed since it is not necessary.

**Comment by ArjixWasTaken** [comment]
> Alright, I think it's all fine as long as it's outside of raylib.js. Please, rename `wasm` folder back and I'll merge this PR. Thank you! 👍

Sure

**Comment by ArjixWasTaken** [comment]
Looks like I modified the permissions of the wasm file, sorry about that, I am using Windows 11 and I can't do much about it.

**Comment by rexim** [comment]
Looks good to me! Thank you! :+1:

**Pull request merged**